### PR TITLE
feat: Overhaul .NET (dotnet) docs

### PR DESCRIPTION
**Pull request by dcramer**
- Break up integrations
- Break up config
- Add dynamic versions
- Add C# documentation as .NET core

### CODE REVIEW

**Comment by dcramer** [comment]
image

image

File: src/platforms/dotnet/guides/aspnetcore/index.mdx [hunk 1]
```diff
@@ -51,6 +32,20 @@ This package extends [Sentry.Extensions.Logging](/platforms/dotnet/microsoft-ext
...
 
+## Installation
+
+Add the Sentry dependency:
+
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.AspNetCore -Version {{ packages.version('sentry.dotnet.aspnetcore') }}
+```
+
+```shell {tabTitle:.Net Core CLI}
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnetcore/index.mdx:43]:
```suggestion
```shell {tabTitle:.NET Core CLI}
```

File: src/platforms/dotnet/guides/extensions-logging/index.mdx [hunk 2]
```diff
@@ -1,35 +1,26 @@
...
-Install-Package Sentry.Extensions.Logging -Version 2.1.5
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.Extensions.Logging -Version {{ packages.version('sentry.dotnet.extensions.logging') }}
 ```
 
-Or using the .NET Core CLI:
-
-```shell
-dotnet add package Sentry.Extensions.Logging -v 2.1.5
+```shell {tabTitle:.Net Core CLI}
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/extensions-logging/index.mdx:17]:
```suggestion
```shell {tabTitle:.NET Core CLI}
```

File: src/platforms/dotnet/guides/serilog/index.mdx [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
-Install-Package Sentry.Serilog -Version 2.1.5
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.Serilog -Version {{ packages.version('sentry.dotnet.serilog') }}
 ```
 
-Or using the .NET Core CLI:
-
-```shell
-dotnet add package Sentry.Serilog -v 2.1.5
+```shell {tabTitle:.Net Core CLI}
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/serilog/index.mdx:17]:
```suggestion
```shell {tabTitle:.NET Core CLI}
```

**Pull request merged**